### PR TITLE
Add jenkins build scripts

### PR DESCRIPTION
**Pull request by RomanKonovalov**
Add jenkins build scripts

### CODE REVIEW

**Comment by RomanKonovalov** [comment]
"retest this please"

**Comment by RomanKonovalov** [comment]
"retest this please"

**Comment by RomanKonovalov** [comment]
"retest this please"

**Comment by RomanKonovalov** [comment]
"retest this please"

**Comment by RomanKonovalov** [comment]
"retest this please"

**Comment by RomanKonovalov** [comment]
"retest this please"

File: commit_build [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+echo "test: yes (jUnit tests for java and javascript, integration tests)"
+echo "code quality checks: yes (findbugs, pmd, checkstyle)"
+echo "deploy: no deploy"
+echo "intent of using: production"
+echo "minification: yes"
```
Comment (UladzimirKapylou) [inline commit_build]: Are you sure that we need `echo` here? I'd like to see this just like comments.

File: commit_build.bat [hunk 2]
```diff
@@ -3,4 +3,4 @@ rem code quality checks: yes (findbugs, pmd, checkstyle)
 rem deploy: no deploy
 rem intent of using: production
 rem minification: yes
-mvn clean verify -Pprod
+mvn clean verify -Pprod %1
```
Comment (UladzimirKapylou) [inline commit_build.bat:8]: What is the intent of using the `%1` parameter?

**Comment by UladzimirKapylou** [comment]
Do we need only one script for Jenkins or this is not final commit in the pull request?

File: commit_build.bat (hunk 2 above)
Comment (RomanKonovalov) [inline commit_build.bat:8]: Using this parameter we can pass argument "-Dprotractor.port=8283" for eighty_develop_builder jenkins job (if we decide to use windows shell)

**Comment by RomanKonovalov** [comment]
This script is suitable for both jenkins jobs

File: commit_build (hunk 1 above)
Comment (RomanKonovalov) [inline commit_build]: Ok. I'll change echo to comment

**Comment by UladzimirKapylou** [comment]
As far as I know, we have three maven jobs on Jenkins.

File: commit_build.bat (hunk 2 above)
Comment (UladzimirKapylou) [inline commit_build.bat:8]: Maybe we need some comment about protractor port and why do we need to change it?

**Comment by RomanKonovalov** [comment]
"retest this please"

**Comment by RomanKonovalov** [comment]
We have 2 maven jobs:   pr_builder and develop builder. Sonar job just invoke maven step

**Comment by UladzimirKapylou** [comment]
Let's use script for it as well. Some additional flags or parameters can be added to it later and it becomes harder to remember and manage them.

**Comment by RomanKonovalov** [comment]
Do you mean to add script to invoke "mvn cobertura:cobertura" task?

**Comment by RomanKonovalov** [comment]
"retest this please"

**Comment by UladzimirKapylou** [comment]
Yes

File: commit_build.bat (hunk 2 above)
Comment (RomanKonovalov) [inline commit_build.bat:8]: done

**Comment by RomanKonovalov** [comment]
"retest this please"

File: commit_build [hunk 3]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+# "test: yes (jUnit tests for java and javascript, integration tests)"
+# "code quality checks: yes (findbugs, pmd, checkstyle)"
+# "deploy: no deploy"
+# "intent of using: production"
+# "minification: yes"
+# "we can modify server port for protractor e2e tests by passing argument "-Dprotractor.port=_port_number_"
```
Comment (UladzimirKapylou) [inline commit_build:7]: Please add a case when we need to do this

Comment (RomanKonovalov) [inline commit_build:7]: done

**Comment by RomanKonovalov** [comment]
"retest this please"

**Comment by UladzimirKapylou** [comment]
:+1:

**Comment by RomanKonovalov** [comment]
"retest this please"

**Comment by SviatlanaHlukhava** [comment]
:+1:

**Pull request merged**